src/js/filterPopup.ts

```diff
@@ -0,0 +1,46 @@
+/* global document, window */
+import setUpPopulationSlider from "./populationSlider";
+import { setUpFilter } from "./filter";
+
+const setUpFilterPopup = (
+  markerGroup,
+  citiesToMarkers,
+  data,
+  searchElement
+) => {
+  const popupElement = document.querySelector(
+    ".filters-popup-window"
+  ) as HTMLElement;
+  const filterIcon = document.querySelector(
+    ".filters-popup-icon"
+  ) as HTMLElement;
+
+  filterIcon.addEventListener("click", () => {
+    popupElement.style.display =
+      popupElement.style.display !== "block" ? "block" : "none";
+  });
+
+  filterIcon.addEventListener(
+    "click",
+    () => {
+      setUpPopulationSlider(markerGroup, citiesToMarkers, data);
+      setUpFilter(markerGroup, citiesToMarkers, data, searchElement);
+    },
+    { once: true }
+  );
```

Comment:
The population wasn't showing because when it was first instantiated, it's trying to grab `offsetWidth`, which is zero if `display: none`. 
One solution was to make the width a variable in setUpPopulationSlider but I think that would make it harder to be responsive to different screens.
The second solution, is to set up the slider when the user first clicks the filter popup. And I ended up moving the filter set up into it too.